cdbot/cogs/cyber.py

```diff
@@ -360,6 +360,72 @@ def __init__(self, bot: Bot):
         await ctx.send(f"There are {len(game_r.members)} that have completed Cyberstart Game. Out of them, "
                        f"{len(true_r.members)} have also completed Essentials and Assess.")
 
+    @command()
+    async def elitecountdown(self, ctx: Context, event_name: str = None, event_level: str = None):
+        """
+        Gets the time until elite, and shows in an embed
+        """
+
+        # Validate and normalise inputs
+        if event_name:
+            event_name = event_name.capitalize()
+            try:
+                event_name = ELITE_ALISES[event_name]
+            except KeyError:
+                content = (
+                    f"{event_name} is not a valid elite location! "
+                    f"Please choose from {', '.join(ELITE_LOCATIONS)}"
+                )
+                return await ctx.send(content)
+
+        if event_level:
+            event_level = event_level[0].lower()
+            if not (event_level in ["o", "y"]):
+                return await ctx.send("Sorry, but that isn't a valid age group! Please choose from 'older', 'younger'")
+
+        # Make array of dates to get
+
+        events_to_get = []
+
+        if event_name:
+            if event_level:
+                events_to_get.append(event_name + event_level)
+            else:
+                events_to_get.append([event_name, "Younger"])
+                events_to_get.append([event_name, "Older"])
+        else:
+            for location in ELITE_LOCATIONS:
+                events_to_get.append([location, "Older"])
+                events_to_get.append([location, "Younger"])
+
+        # Get dates and send
+
+        events = []
+        for event in events_to_get:
+            events.append([event[0], event[1], self.get_days_and_months_until(ELITE_DATE_KEYS[event[0]][event[1]])])
+
+        # Display to user
+
+        if len(events) == 1:
```

Comment:
This doesn't work. This command got no reply:
![image](https://user-images.githubusercontent.com/3729048/59565973-7b7c3880-9051-11e9-80b7-67b5815b5dde.png)